packages/react-native-renderer/src/__tests__/ReactFabricAndNative-test.internal.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
+'use strict';
+
+let React;
+let ReactFabric;
+let ReactNative;
+let createReactNativeComponentClass;
+let UIManager;
+let FabricUIManager;
+
+describe('ReactFabric', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    ReactNative = require('react-native-renderer');
+    jest.resetModules();
+    jest.mock('shared/ReactFeatureFlags', () =>
+      require('shared/forks/ReactFeatureFlags.native-fabric'),
+    );
+
+    React = require('react');
+    ReactFabric = require('react-native-renderer/fabric');
+    FabricUIManager = require('FabricUIManager');
+    UIManager = require('UIManager');
+    createReactNativeComponentClass = require('../createReactNativeComponentClass')
+      .default;
+  });
+
+  it('find Fabric nodes with the RN renderer', () => {
+    const View = createReactNativeComponentClass('View', () => ({
+      validAttributes: {title: true},
+      uiViewClassName: 'View',
+    }));
+
+    let inst;
+
+    class Component extends React.Component {
+      render() {
+        inst = this;
+        return <View title="foo" />;
+      }
+    }
+
+    ReactFabric.render(<Component />, 11);
```

Comment:
let inst =?

Reply:
I think we wanted to get rid of that at some point. I should probably use a ref though.